components/_global-navigation.scss

```diff
@@ -153,6 +157,18 @@ $notification-counter-size: 18px;
 
 	// End of temporary styles
 
+	&__links-and-search {
+		display: flex;
+		flex: 4 1 auto;
+	}
+
+	&__link,
+	&__wikis-menu {
+		max-width: 100%;
+		transition-duration: $search-transition-duration;
+		transition-property: color, flex, font-size, max-width, padding;
```

Comment:
When there's multiple properties to be animated in one duration it is much more DRY and readable to write it this way. In this case it would look like below. :dizzy_face:
```
transistion: color $search-transition-duration, flex $search-transition-duration, font-size $search-transition-duration, max-width $search-transition-duration, padding $search-transition-duration;
```